matchers.go

```diff
@@ -57,6 +57,11 @@ func HTTP1Fast(extMethods ...string) Matcher {
 	return PrefixMatcher(append(defaultHTTPMethods, extMethods...)...)
 }
 
+// TLS matches HTTPS requests.
+func TLS() Matcher {
```

Comment:
Good idea. I'm not terribly familiar with the protocol, so if you know of any good docs I can take a stab.

This simple version worked well enough for my needs, so that's what I ended up using.